File: src/nomanssave/ct.java
package nomanssave;

import java.io.File;
import javax.swing.Icon;
import javax.swing.filechooser.FileView;

class ct extends FileView {
   // $FF: synthetic field
   final cs fP;

   ct(cs var1) {
      this.fP = var1;
   }

   public Icon getIcon(File var1) {
      String var2 = var1.getName();
      return (Icon)(var2.endsWith(".fb3") ? cs.as() : super.getIcon(var1));
   }

   public String getName(File var1) {
      String var2 = var1.getName();
      return var2.endsWith(".fb3") ? var2.substring(0, var2.length() - 4) : var2;
   }
}

File: NMSSaveEditor/nomanssave/lower/ct.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.IO;
using System.Linq;
using System.Text;
using System.Text.RegularExpressions;
using System.Windows.Forms;
using System.Globalization;

namespace NMSSaveEditor
{

public class ct : FileView {
   // $FF: synthetic field
   cs fP;

   ct(cs var1) {
      this.fP = var1;
   }

   public Image getIcon(FileInfo var1) {
      string var2 = var1.Name;
      return (Icon)(var2.EndsWith(".fb3") ? cs.as() : base.getIcon(var1));
   }

   public string getName(FileInfo var1) {
      string var2 = var1.Name;
      return var2.EndsWith(".fb3") ? var2.Substring(0, var2.Length - 4) : var2;
   }
}

}
